### DISCUSSION TITLE
enable extraAttributes() on Grid

### DISCUSSION THREAD

**Discussion by tobyhage** [filamentphp/filament#2139] [Help] [answered]
Hello,

On this moment extraAttributes() on Grid is not working. 
The HasExtraAttributes concern is available but not implemented in the grid component view.

Can please somebody enable extraAttributes() on Grid.

I have looked in the code to make a PR, but i don't understand how to do it :-(

**Comment by danharrin** [comment, marked as answer]
bbfaa936619a1dd1a4c106f4c2a2736ff6ad7261

**Reply by tobyhage** [reply]
Thank you very much!!
